app/models/bot.rb

```diff
@@ -12,8 +12,36 @@ class Bot < ActiveRecord::Base
   validates :hair_color, presence: true
   validates :user_id, presence: true
 
+  WELCOME_MESSAGE = 'Yeiii!'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.